releasenotes/notes/0.19/symengine-bump-20dedd5204870e7a.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The minimum version of Symengine required for installing has been increased
```

We probably should add a link to it on pypi:
```suggestion
    The minimum version of `Symengine <https://pypi.org/project/symengine/>`__ required for
    installing has been increased
```

---

Done in 33ffd86f4.